draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -526,11 +524,11 @@ To send RTP/RTCP packets over QUIC streams, a sender MUST open at least one new
 RoQ uses unidirectional streams, because there is no
 synchronous relationship between sent and received RTP/RTCP packets. A peer that
 receives a bidirectional stream with a flow identifier that is associated with
-an RTP or RTCP stream, SHOULD stop reading from the stream and send a
+an RTP or RTCP stream, MUST stop reading from the stream and send a
```

Comment:
I wonder if this should lead to closing the connection because it is a protocol violation? We make that change in a different PR though.

Reply:
Actually, you're right - we talked about that, but somehow it didn't make the transition into our shared notes from our call. Let's wait to merge until I can update the PR (I'm having odd Windows problems right now, so that may take a day or two).

Reply:
@mengelbart - I think I got this one right. Please let me know! 

If I did, we should be able to merge the PR.